### ISSUE TITLE
Bounty Creation - Show as Organization 

### ISSUE THREAD

**Issue by GTChase** [gitcoinco/web#10390]
**User Story:**
As a funder 
I want to be able to show my Github organization as the bounty creator
So that the bounty is tied to my org and not my individual account

**Acceptance Criteria**
GIVEN I am creating a bounty
WHEN i navigate to the bounty details step
THEN there will be a field labeled show as organization
AND the user should be able to enter a github org URL
AND this field should be optional

**Product & Design Links:**
https://www.figma.com/file/8AFLrc4dx9xnMXUkWOAaAR/Existing-Bounties-Updates?node-id=196%3A3433

**Notes/Assumptions:**

**Comment by nutrina** [comment]
The field exists in the UI, but it is not clear where and how to use it?

**Comment by GTChase** [comment]
This field is automatically being bordered with red, indicating to the user that it is required. this field should be optional, so could we remove the red border always? The field is acting as optional otherwise, so once we get that removed we should be good! @nutrina 
<img width="821" alt="Screen Shot 2022-04-21 at 3 44 19 PM" src="https://user-images.githubusercontent.com/95642312/164540559-8854b557-7f95-4a3a-83bf-a9b36e08718e.png">

**Comment by willsputra** [comment]
yep other than removing the red border as @GTChase mentioned - lgtm!
